pymc/gp/cov.py

```diff
@@ -812,6 +824,52 @@ def full(self, X, Xs=None):
     def diag(self, X):
         X, _ = self._slice(X, None)
         return self.cov_func(self.w(X, self.args), diag=True)
+    
+
+class WrappedPeriodic(Covariance):
+    r"""
+    Wrap a stationary covariance function to make it periodic.
+
+    This is done by warping the input with the function
+
+    .. math::
+        \mathbf{u}(x) = \left(
+            \mathrm{sin} \left( \frac{2\pi x}{T} \right),
+            \mathrm{cos} \left( \frac{2\pi x}{T} \right)
+        \right)
+
```

Comment:
It might be nice to add something like, "the `GeneralizedPeriodic` kernel constructs periodic kernels from any `Stationary` kernel"

Also, I think it'd be nice to add a note that describes and gives the code that makes this function equivalent to `Periodic`, but mention in that case using that `Periodic` might be a bit faster.  

Also, the function $u(x)$ is defined, but without context I'd have to know where to look this up.  Could you point to a reference or maybe add a bit more detail here (or both)?

Reply:
Have addressed these in latest commit.

Reply:
Thank you!  Super nice